debezium-core/src/main/java/io/debezium/transforms/ExtractNewRecordStateConfigDefinition.java

```diff
@@ -20,7 +20,8 @@ public enum DeleteTombstoneHandling implements EnumeratedValue {
         DROP("drop"),
         TOMBSTONE("tombstone"),
         REWRITE("rewrite"),
-        REWRITE_WITH_TOMBSTONE("rewrite-with-tombstone");
+        REWRITE_WITH_TOMBSTONE("rewrite-with-tombstone"),
+        REWRITE_DELETES("rewrite-deletes");
```

Comment:
@Naros I wonder if the option name is not a bit misleading. Should not `delete-to-tombstone` or `delete-as-tombstone` be better option?

Reply:
@jpechane / @vaibhav-yb I agree and I believe I'd prefer `delete-to-tombstone`, but ymmv.